horace_core/sqw/@axes_block/private/normalize_bin_input_.m

```diff
@@ -9,17 +9,18 @@
 % ------------
 % grid_size -- the size of the grid, the pixels will be rebinned on
 %
-% pix_coord -- [3,npix] or [4,npix] numeric array of the pixel coordinates
+% pix_coord -- [3,npix] or [4,npix] or [43] numeric array of the pixel
+%               coordinates. If
 % mod       -- operation mode specifying what the following routine should
```

Comment:
Why shorten `mode` to `mod`? `mod` means many different things to me, but never `mode`

Reply:
to be fair, not worth bothering, but changed